src/Model/Repository/MyParcelConsignmentRepository.php

```diff
@@ -569,6 +571,18 @@ private function encodeInsurance()
         return $this;
     }
 
+    private function encodePhysicalProperties()
+    {
+        if (empty($this->getPhysicalProperties()) && $this->getPackageType() != 4) {
+            return $this;
+        }
+        if ($this->getPackageType() == self::PACKAGE_TYPE_DIGITAL_STAMP && !isset($this->getPhysicalProperties()['weight'])) {
```

Comment:
```suggestion
        if (!isset($this->getPhysicalProperties()['weight'])) {
```
Is `getPackageType() == self::PACKAGE_TYPE_DIGITAL_STAMP` nodig?